main/EDPftvarcon.F90

```diff
@@ -49,6 +49,7 @@ module EDPftvarcon
                                                      ! that is occupied by crown. For fire model. 
      real(r8), allocatable :: bark_scaler(:)         ! scaler from dbh to bark thickness. For fire model.
      real(r8), allocatable :: crown_kill(:)          ! scaler on fire death. For fire model. 
+     real(r8), allocatable :: active_crown_fire(:)   ! Is plant susceptible to active crown fire?
```

Comment:
Depending on how active_crown fire is implemented, this may be altered to a site level "flag" rather than a PFT level parameter. @lmkueppers @slevisconsulting @pollybuotte we should discuss on Friday.
@rgknox can I put a hold on this until we have that discussion?

Reply:
@jkshuman I think your question can be summed up as this:
Will we use this flag as an active crown fire on/off switch for all pfts simultaneously?
OR
Will we use it at the pft level to assign varying pft vulnerability to active crown fire?

Reply:
I prefer to keep the current flexibility of pft-level and am open to discussing.

Reply:
what are the arguments pro and con?

Reply:
Pro for keeping pft-level:
The flexibility to say that, for example, ponderosa pine is more vulnerable to active crown fire than sugar maple by assigning different values between 0 and 1 to each pft.

No cons in my opinion, because we can keep using the pft-level flag as a simple on/off switch if we prefer.

Reply:
@jkshuman and @slevisconsulting , thanks for the feedback, @jkshuman does this sound reasonable, or do you feel like we should take the time now to swap to non-pft parameter here?

Reply:
I see no reason we can't wait to here back from your discussion on Friday, take your time.

Reply:
This is really a result of the transition from cohort-level implementation to patch-level, and the transition to calculating scorch height at the cohort-level.
Cons: how to justify PFT level implementation of a stand level behavior
Pros: ?

This is more a reflection of how crown fire happens in the stand at the patch level (so patch level not site-level). For the patch, you get active crown fire if you meet two conditions: 
1) there is enough fuel in the canopy to achieve passive crown initiation to propagate fire vertically through the canopy,  (fire intensity from canopy fuels >=  surface fire intensity)
and 2) the fire can spread from crown to crown (fire intensity using canopy rate of spread >= surface fire intensity)

Both of these conditions use patch level stand values from the total canopy, so controlling active crown fire at the PFT level may not be valid. The alpha_SH parameter already acts as a PFT level control on crown scorch, so I think working within the parameter space makes sense. This makes more sense with the transition to calculating scorch height at the cohort-level implemented in PR #594 and carried over into #584 and #572
                `                !Equation 16 in Thonicke et al. 2010 !Van Wagner 1973 EQ8 !2/3 Byram (1959)
                currentCohort%Scorch_ht = EDPftvarcon_inst%fire_alpha_SH(currentCohort%pft) * (currentPatch%FI**0.667_r8)` 


This does depend on the implementation of active crown fire and assumptions about patch size. In a model with a fixed patch size of "appropriate" size, when active crown fire ignites assuming 100% crown damage is reasonable. For FATES  this assumption may not make sense, as the area of application can be quite large. With implementation of a feedback where the fire intensity and behavior is recalculated with active fire, this would alter the scorch height and allow for use of the altered (higher) value. With large areas as @adrifoster noted in #573 stand density and wind are factors: >  "Spotting fires could bridge large spaces between tree crowns but this requires very high fire intensities (30,000 - 100,000+ kW/m). "

This comes back to using the existing PFT level alpha_SH value and recalculating fire intensity and behavior for the patch/stand.  Prior to implementing the feedback onto fire intensity and behavior, using 100% crown damage on the patch for active crown fire.

Reply:
Hi All,  we decided on our call today that this "active_crown_fire" flag should not be a PFT-level parameter, but should instead be either a global flag or a namelist variable (whichever is easier to write/maintain).  We just need to be able to turn it on or off for a given model run.

Reply:
thanks for the update.  I will add this to the parameter file.